data_dir/ssl_conf/db.crt

```diff
@@ -1,33 +1,33 @@
 -----BEGIN CERTIFICATE-----
-MIIFoDCCA4gCCQDbIeSUGamu9DANBgkqhkiG9w0BAQUFADCBkTELMAkGA1UEBhMC
-U0UxEjAQBgNVBAgTCVN0b2NraG9sbTESMBAGA1UEBxMJU3RvY2tob2xtMRAwDgYD
-VQQKEwdmb28uYmFyMRAwDgYDVQQLEwdmb28uYmFyMRMwEQYDVQQDEwpkYi5mb28u
-YmFyMSEwHwYJKoZIhvcNAQkBFhJwb3N0bWFzdGVyQGZvby5iYXIwHhcNMTkwODA2
-MTAwNTI3WhcNMjEwMTAxMTAwNTI3WjCBkTELMAkGA1UEBhMCU0UxEjAQBgNVBAgT
-CVN0b2NraG9sbTESMBAGA1UEBxMJU3RvY2tob2xtMRAwDgYDVQQKEwdmb28uYmFy
-MRAwDgYDVQQLEwdmb28uYmFyMRMwEQYDVQQDEwpkYi5mb28uYmFyMSEwHwYJKoZI
-hvcNAQkBFhJwb3N0bWFzdGVyQGZvby5iYXIwggIiMA0GCSqGSIb3DQEBAQUAA4IC
-DwAwggIKAoICAQDlm2dpBfpJR3NtAfkYFAPmfPhpYVJluKnFbjlUEB0INJARU6Uh
-tpCNS5PkpKfQkjtMxNgu7JfreuGh/wKgGcFyzgA4aEDD6LuHaNrvQbMnAatoVJ3e
-+GCmRsBlCRM7sYCaoiVfithPehnaRO1+UK3JJVos+4FwanTa2cmF3jJbVjs/FSgy
-fevYtjvyf66E0rOe6xoPJjPb8vgfTxHl4R2ScHEaeKpKu7ZLLGqOIbOCiTp0br+b
-5o2EP5Uwica0Pnou2GkNcL8v+qCaAft9L/31hs7SnWR2qSvTGyqX8Uz0Od7uDT8j
-TryjPVqFXIf7hQCeq/6VkRhU7KtdQzmDF78cqylg+64Ze780BnXodFo1zghEKGHO
-OYjOARbWnu5qcMPXrSrAwkqq3xfui5zMFjWrvLX2EF7ygTuoMYw2sqbE3RHhtY+b
-aIgu7OuXZKAF8sGTneGbiEBzUGD/8hHXLJAJqQ6TKue5za9XeBxVmOZf5qEsAEYZ
-+19QDZbbWLkUytVEUAxnVw1wb9e9EZcU9p2vQYk3EyoRunxHYT++mRVlopQeaiew
-FOEj29l5eaD9B9hOCrMFAnOM5pblglKow/ygEASW+Ttg0M7GejnmAWHXQoQjwNGQ
-MV+GLMa94uU++LqodZMPu2VmDjIdDq5P181jToTLWqSknzLBSboZKkIkFwIDAQAB
-MA0GCSqGSIb3DQEBBQUAA4ICAQBLrwf09fKwD8SF1um/i22bNq2pagkYZzIBlM6v
-MMFGfh1wXZ7AWiXnt1LyvVdR5QUeIRdGWgKRk2bmeLOVc+toLfKcX/Hj6aZswa/v
-mY/JqyEhhJ/KFgWlla2kZS5L3O39gwzLigcAWSdqxurx+i/SzPLllS+vc1ixRXFP
-KMG80Tj6e0lCuUcUlYiMABXurrXkZ+fXBgT9mycHU8dVYqpj2q/OCov1ylzFg9qw
-ymLMnlJ2egK+sdmounPc9WLGPp7h8rj7SapAHNqdrrDMS80ecEDyB2c2fJiVddd1
-sBhfNvulQ3mwys+EFyKMcXqW5SgFq/laVVyXjkaEIVuDAXcuNBFNwKRq9Xy596fh
-hcc50eq0EyEg4OATAbD/XvMi3G+ALoamgRauBX7Lj/sCu4vQIihBmcbC36CnFOGd
-Jz46vHD9MWXq3d8Xs7MyZIgAFpOeyUYlt67TLFy7vR/TlbVOi0kwT8fcDnVfRcIy
-T/y/7qy/1mStMKKCfTauJnuMcWLcw0PTwOklgRieY9EZKlNJ2AxTlY4g3tEK3znJ
-94SKeUqiGh7/7YHXIHidLswMp+Se5IH5jk5RiCWgDVJehbskRiKuz/8I7wdqkbmJ
-rmsVlHLEpy7n6AMaq4FMRAGau6vdM4AF5a1QdnTtNGSxFJ9SqLsnbdj/xlPCxl8x
-1EoTNQ==
+MIIFqzCCA5MCFHRqkuSRRt5/NTbEYq7vCtmotoB+MA0GCSqGSIb3DQEBCwUAMIGR
```

Comment:
This file had been deleted from SCT repo.

Reply:
yeah I know